src/strat_engine/pool.rs

```diff
@@ -56,10 +65,24 @@ impl Pool for StratPool {
     }
 
     fn add_blockdev(&mut self, path: &Path) -> EngineResult<()> {
-        let bd = try!(BlockDev::initialize(&self.uuid, &[path], MIN_MDA_SIZE, true))
-            .pop()
+        let dev = try!(Device::from_str(&path.to_string_lossy()));
```

Comment:
Please file an issue about the problem alluded to in the commit message. It seems bad that pools can steal devs from each other.